lib/web_ui/lib/ui_web/src/ui_web/plugins.dart

```diff
@@ -0,0 +1,22 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:typed_data';
+
+import 'package:ui/src/engine.dart';
+import 'package:ui/ui.dart' as ui;
+
+typedef PluginMessageHandler = Future<void> Function(
```

Comment:
Does this need a dartdoc?

Reply:
I removed this. We already have the same typedef in `dart:ui` so I used that instead.